core/vm/ovm_state_manager.go

```diff
@@ -210,6 +210,20 @@ func testAndSetContractStorage(evm *EVM, contract *Contract, args map[string]int
 	return []interface{}{true}, nil
 }
 
+func hasEmptyAccount(evm *EVM, contract *Contract, args map[string]interface{}) ([]interface{}, error) {
```

Comment:
Original contract function is here for reference:

https://github.com/ethereum-optimism/contracts-v2/blob/7e82fce506df729dfd070e0205d98c1adc9e6dcd/contracts/optimistic-ethereum/OVM/execution/OVM_StateManager.sol#L185-L199